apisix/upstream.lua

```diff
@@ -235,7 +235,8 @@ function _M.set_by_route(route, api_ctx)
 
         local dis = discovery[up_conf.discovery_type]
         if not dis then
-            return 500, "discovery " .. up_conf.discovery_type .. " is uninitialized"
+            local err = "discovery " .. up_conf.discovery_type .. " is uninitialized"
```

Comment:
Why do we need this style?

Reply:
Avoid line too long

Reply:
The same length is more confusing. 

![image](https://user-images.githubusercontent.com/6814606/125878703-0221fb4d-8972-4d6a-9089-218ba1e4e28f.png)